src/city/data.c

```diff
@@ -593,6 +598,11 @@ static void load_main_data(buffer *main)
     city_data.migration.refused_immigrants_today = buffer_read_i32(main);
     city_data.migration.percentage = buffer_read_i32(main);
     city_data.culture.population_with_venus_access = buffer_read_i32(main);
+    city_data.culture.average_theater = buffer_read_i32(main);
```

Comment:
Adding these buffers will require bumping the savegame version and also adding compatibility for loading previous versions which don't have these values.

It's more or less like the building data changes, but let me know if you need any help with this.